ADALiOS/ADALiOS/ADAuthenticationResult.h

```diff
@@ -54,14 +54,15 @@ typedef enum
  case of error.*/
 @property (readonly) NSString* accessToken;
 
+/*! A valid access token, if the results indicates success. The property is
+ calculated from the tokenCacheStoreItem one. The property is nil, in
+ case of error.*/
+@property (readonly) NSString* idToken;
```

Comment:
Rename accesstoken to token

Sent from Outlook<http://aka.ms/Ox5hz3>




On Wed, Sep 16, 2015 at 4:10 PM -0700, "Brandon Werner" <notifications@github.com<mailto:notifications@github.com>> wrote:


In ADALiOS/ADALiOS/ADAuthenticationResult.h<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzureAD%2fazure-activedirectory-library-for-objc%2fpull%2f378%23discussion_r39696516&data=01%7c01%7cKanishk.Panwar%40microsoft.com%7c76bf467bf2f44abdbe8308d2beebf419%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=9jW7vMWJ6f6597ht98i%2ffXWOHJKU1li6sse5S6unp54%3d>:

> @@ -54,14 +54,15 @@ typedef enum
>   case of error.*/
>  @property (readonly) NSString* accessToken;
>
> +/*! A valid access token, if the results indicates success. The property is
> + calculated from the tokenCacheStoreItem one. The property is nil, in
> + case of error.*/
> +@property (readonly) NSString* idToken;


This was a prototype property that can be removed. I now just assign the idToken to the accessToken if we are in a flow for idToken only.

removed.

-
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzureAD%2fazure-activedirectory-library-for-objc%2fpull%2f378%2ffiles%23r39696516&data=01%7c01%7cKanishk.Panwar%40microsoft.com%7c76bf467bf2f44abdbe8308d2beebf419%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=YWLLHddEA7xPk5xj9biw7BSbVgYcrfVGY4%2fe8Kyz87U%3d>.